quantum/quantum.h

```diff
@@ -230,6 +230,10 @@ extern layer_state_t layer_state;
 #    include "pointing_device.h"
 #endif
 
+#ifdef POINTING_DEVICE_AUTO_MOUSE_ENABLE
+#    include "pointing_device_auto_mouse.h"
+#endif
+
```

Comment:
This isn't needed, as pointing_device.h already includes this, and is included right above.  So it can be safely removed.
```suggestion
```